flink/flink1.10-shims/pom.xml

```diff
@@ -33,7 +33,7 @@
     <name>Zeppelin: Flink1.10 Shims</name>
 
     <properties>
-        <flink.version>1.10.0</flink.version>
```

Comment:
does it make sense to define a constant in the `flink-parent` so it will be shared?

Reply:
I am afraid not. I plan to add profile flink-1.10 and flink-1.11 for flink interpreter after flink 1.11 is released. But `flink.version` in `flink1.10-shims` will not be affected by profile, it should always be flink 1.10, `flink.version` in `flink/interpreter` module will be affected by this profile, and will run test under `flink/interpreter` for different flink version.

Reply:
I meant, that we can define property, like, `<flink.1.10.version>` and refer to it... I did this for Hadoop:

```
    <hadoop2.7.version>2.7.7</hadoop2.7.version>
    <hadoop2.6.version>2.6.5</hadoop2.6.version>
    <hadoop3.0.version>3.0.3</hadoop3.0.version>
    <hadoop3.1.version>3.1.3</hadoop3.1.version>
```

and then use it as `${hadoop3.1.version}`, etc.

Reply:
Good idea, I have fixed it